pkg/inventory/misc.go

```diff
@@ -0,0 +1,17 @@
+package inventory
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// initK8sObjectMeta will set the required default settings to
+// kubernetes objects metadata if is required.
+func initK8sObjectMeta(obj metav1.Object) {
```

Comment:
Could you move this to `pkg/util/util.go`?

Also would be good to have a test for this.